shared/src/business/useCases/generatePractitionerCaseListPdfInteractor.ts

```diff
@@ -34,13 +34,17 @@ export const generatePractitionerCaseListPdfInteractor = async (
 
   const { barNumber, name } = practitionerUser;
 
-  const cases = await applicationContext
+  const docketNumbers = await applicationContext
     .getPersistenceGateway()
-    .getCasesAssociatedWithUser({
+    .getDocketNumbersByUser({
       applicationContext,
       userId,
     });
 
+  const cases = await applicationContext
+    .getPersistenceGateway()
+    .getCasesByDocketNumbers({ applicationContext, docketNumbers });
+
```

Comment:
Could use OpenSearch here, but trying to avoid it if not necessary

Reply:
Hmm.. in my mind, it's 1 opensearch call vs 2 dynamodb calls to get a subset of a case. It's a nit for me but I prefer using opensearch for fetching cases in this instance.